packages/handbrake.rb

```diff
@@ -94,20 +94,15 @@ def self.install
     Dir.chdir 'x86_64-cros-linux-gnu' do
       system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     end
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d"
+    FileUtils.ln_s "#{CREW_PREFIX}/bin/HandBrakeCLI", "#{CREW_DEST_PREFIX}/bin/hb"
```

Comment:
Doesn't this draw from the current install instead of the files in `CREW_DEST_PREFIX`?

Reply:
Also, why are we creating this symlink?

Reply:
The first argument is the source; the second argument is the target.  The target is what we want to create so that when it installs, it has the correct path.